packages/ovh-manager/ovh-manager.js

```diff
@@ -92,4 +99,46 @@ angular
     _.set(OvhHttpProvider, 'returnSuccessKey', 'data'); // By default, request return response.data
     _.set(OvhHttpProvider, 'returnErrorKey', 'data'); // By default, request return error.data
   })
-  .config(routing);
+  .config(routing)
+  .service('ManagerNavbarService', navbarService)
```

Comment:
You should declare a scoped module for that :)

This first module should contain only config and import I think :)